deploy/install/01-prerequisite/03-crd.yaml

```diff
@@ -350,23 +351,12 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.7.0
+    controller-gen.kubebuilder.io/version: v0.8.0
+  creationTimestamp: null
   labels:
     longhorn-manager: ""
   name: backingimages.longhorn.io
 spec:
-  conversion:
-    strategy: Webhook
-    webhook:
-      clientConfig:
-        service:
-          name: longhorn-conversion-webhook
-          namespace: longhorn-system
-          path: /v1/webhook/conversion
-          port: 9443
-      conversionReviewVersions:
-      - v1beta2
-      - v1beta1
```

Comment:
Can you explain why are we removing these?

Reply:
Hmm, I already checked all my command versions in [generate_code.sh](https://github.com/longhorn/longhorn-manager/blob/master/k8s/generate_code.sh). I am not sure why this is removed. Could you help to run the script again to see whether this is added back? Thank you.